style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: hsl(207, 36%, 49%);
+  text-align: center;
+}
+
+h1 {
+  color: hsl(343, 77%, 87%);
+  font-size: 50px;
+  font-family: "Baskerville", serif;
```

Comment:
While you include `font-family` declarations in your CSS, you are relying on a user's system fonts and not using [web fonts](https://mica-web.github.io/learn/css/fonts#web-fonts), which is one of the requirements for this project.